src/ComplexMethod.java

```diff
@@ -1,8 +1,141 @@
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
-
 public class ComplexMethod {
 
+    public String iasdntToEnglishValue(int i) {
+        /*dfg dfgdsf gdsfg dsfg sdf gdsgds*/
+        if (i == 1) {
+            return "One";
+        }
+        if (i == 2) {
+            return "Two";
+        }
+        if (i == 3) {
+            return "Three";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Three" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)